go.mod

```diff
@@ -27,18 +27,27 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require (
+	github.com/devfile/alizer v1.2.2-0.20231004141146-f36141673c7f // indirect
```

Comment:
This should be in the already existing require section for indirect dependencies.

Reply:
I'll fix it later